src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -770,35 +987,37 @@ internal void SendUndoToTutorial(TutorialState tutorial)
         tutorial.EditorUndoTutorial.EditorUndoButtonControl = undoButton;
     }
 
-    private static void SetOrganelleButtonStatus(Button organelleItem, bool nucleus)
+    private static void SetOrganelleButtonStatus(Control organelleItem, bool nucleus)
     {
-        if (organelleItem.Name == "nucleus")
+        var button = organelleItem.GetNode<Button>("VBoxContainer/Button");
+
+        if (organelleItem.Name == "Nucleus")
         {
-            organelleItem.Disabled = nucleus;
+            button.Disabled = nucleus;
         }
-        else if (organelleItem.Name == "mitochondrion")
+        else if (organelleItem.Name == "Mitochondrion")
         {
-            organelleItem.Disabled = !nucleus;
+            button.Disabled = !nucleus;
         }
-        else if (organelleItem.Name == "chloroplast")
+        else if (organelleItem.Name == "Chloroplast")
         {
-            organelleItem.Disabled = !nucleus;
+            button.Disabled = !nucleus;
         }
-        else if (organelleItem.Name == "chemoplast")
+        else if (organelleItem.Name == "Chemoplast")
         {
-            organelleItem.Disabled = !nucleus;
+            button.Disabled = !nucleus;
         }
-        else if (organelleItem.Name == "nitrogenfixingplastid")
+        else if (organelleItem.Name == "Nitrogen Fixing Plastid")
         {
-            organelleItem.Disabled = !nucleus;
+            button.Disabled = !nucleus;
         }
-        else if (organelleItem.Name == "vacuole")
+        else if (organelleItem.Name == "Vacuole")
         {
-            organelleItem.Disabled = !nucleus;
+            button.Disabled = !nucleus;
         }
-        else if (organelleItem.Name == "oxytoxy")
+        else if (organelleItem.Name == "Toxin Vacuole")
```

Comment:
Here's a good example why InternalName exists, it is meant to always stay the same even when we rename stuff like oxytoxy to "Toxin Vacuole" code shouldn't break.